### ISSUE TITLE
Schematic symbols are flipped vertically when no rotation is there

### ISSUE THREAD

**Issue by imrishabh18** [tscircuit/circuit-to-svg#125]
Test file - https://github.com/tscircuit/core/blob/main/tests/components/normal-components/potentiometer.test.tsx
![Image](https://github.com/user-attachments/assets/2a08f103-8369-4cc1-95a6-e49161638189)

Issues to fix
- schRotation 180 and 0 are same
- schematic-symbols and schRotaion 0, if you compare is vertically flipped which should not be the case

![Image](https://github.com/user-attachments/assets/2066fa8e-bc4e-44f8-90a1-9c6b842d9ae1)

**Comment by anas-sarkez** [comment]
@imrishabh18 
I'm already Implemented functions might help you 
-flipSymbolOverXAxis
-flipSymbolOverYAxis
in Schematic symbols repo

**Comment by imrishabh18** [comment]
@anas-sarkez This issue was regarding finding out why the `schematic-symbol` is flipped over X axis in `circuit-to-svg` compared to how it's shown in the.

Cause if the user goes to the `schematic-symbols` directory and see's the symbol, then the schematic view should replicate that on usage. Only when the rotation is added to props, then we can probably use the functions which you have implemented.

**Comment by seveibar** [comment]
@imrishabh18 its possible they are flipped in schematic symbols. Somebody please check your data before flipping!

**Comment by imrishabh18** [comment]
@seveibar Yeah, could be. I was seeing the directory of symbols as source of truth on appearance. So, when I imported that to the circuit to svg, it appeared flipped.
But true, we can take a deeper look to understand.
